tenks.yml

```diff
@@ -9,7 +9,7 @@ node_types:
     volumes:
       # There is a minimum disk space capacity requirement of 4GiB when using Ironic Python Agent:
       # https://github.com/openstack/ironic-python-agent/blob/master/ironic_python_agent/utils.py#L290
-      - capacity: 20GiB
+      - capacity: 40GiB
```

Comment:
Do you know how much disk actually gets used? We generally try to keep the footprint from growing too large if possible.

Reply:
We bump this for some workshops, see for example https://github.com/stackhpc/a-universe-from-nothing/commit/db2156dfa554683799fac48475236e20a5c715b0 in the wallaby-ovn-monitoring branch.

Maybe we could keep a few different tenks.yml files in the home directory for different configurations?

Reply:
> Do you know how much disk actually gets used? We generally try to keep the footprint from growing too large if possible.

@markgoddard after having left the instance running for a few weeks, the entire 20GB was used up



> We bump this for some workshops, see for example [db2156d](https://github.com/stackhpc/a-universe-from-nothing/commit/db2156dfa554683799fac48475236e20a5c715b0) in the wallaby-ovn-monitoring branch.
> 
> Maybe we could keep a few different tenks.yml files in the home directory for different configurations?

@priteau Perhaps. If we change it for some workshops where the workshop would span over multiple sessions then perhaps we don't need to make this change?

Reply:
I'm not against increasing it, just wondering whether doubling is going too far